containers/ddev-webserver/Dockerfile

```diff
@@ -25,6 +25,7 @@ ENV COMPOSER_ALLOW_SUPERUSER 1
 ENV COMPOSER_CACHE_DIR /mnt/ddev-global-cache/composer
 # Windows, especially Win10 Home/Docker toolbox, can take forever on composer build.
 ENV COMPOSER_PROCESS_TIMEOUT 2000
+ENV COMPOSER_BIN_DIR $WEBSERVER_DOCROOT/vendor/bin
```

Comment:
I'd rather see all the changes just be in the skel .profile. I don't think there's any reason not to.